modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,17 @@
+= AeroGear Processes
+
+image::aerogear_logo_200px.png[]
+
+This documentation covers the AeroGear processes relating to:
+
+* User Feedback
```

Comment:
So there could be an additional heading here?

Reply:
@laurafitzgerald  we can add whatever we want here, as you said the intention here is to just set up a skeleton